.github/workflows/integration-tests.yml

```diff
@@ -609,3 +609,32 @@ jobs:
           ${SCCACHE_PATH} --show-stats
 
           ${SCCACHE_PATH} --show-stats | grep -e "Cache hits\s*[1-9]"
+
+  issues-1652:
```

Comment:
Could you please rename the test for something more explicit?

Reply:
do you have any prefer name? is missing-debug-info a good name?